src/app/meetups/meetupsadd.component.ts

```diff
@@ -16,7 +16,8 @@ import { CouchService } from '../shared/couchdb.service';
         </div>
         <button name="singlebutton" class="btn btn-primary" type="submit">Save</button>
     </form>
-    `
+    `,
+    providers:[CouchService]

```

Comment:
Same comment as above, please move this into the `meetupsadd.component.spec.ts` file